assets/css/password-requirements.css

```diff
@@ -0,0 +1,12 @@
+/**
+ * @file
+ * Minimal style for the password requirements field.
+ */
+
+#password-policy-constraints [data-validation-passed="no"] {
```

Comment:
If I know right you can override the default style by using the `#password-policy-constraints .constraint[data-validation-passed="no"` selector, no need for `!important`.

The template can be overridden, so you can add classes if you want, please check the `password-enhancements-policy-constraints.html.twig` file.

Reply:
You can override like that, but you can't override by adding a `.validation-passed` class without using important, as the `ID child` selector will be stronger. Unless it's the last resort, when theming you try not to do `#someid .someclass` or `#someid someelement` selections because then you can't use classes for the child element.

Reply:
I see the issue with that, but otherwise I cannot ensure that it is not going to jump on another element with the same attribute and value if any other element would have the same, IOW the attribute selector is not unique.

Reply:
I don't really have better suggestions, I'd add a helper class when the `data-validation-passed` changes depending on it's value that is specific enough like `password-enhancements-validation-passed`, then there's a big chance the target is specific enough and it's also easy enough to override it.

From a theming perspective it would be also cool if this class could be changed from the ui or some preprocess/alter function.

The best practice is to use classes for theming, so I wouldn't consider it redundant especially if it's configurable.